user/serializers.py

```diff
@@ -1 +1,30 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    user_class = get_user_model()
+
+    class Meta:
+        model = get_user_model()
```

Comment:
why duplicated again?...
```
class UserSerializer(serializers.ModelSerializer):
    user_class = get_user_model()

    class Meta:
        model = user_class 
```

Reply:
yeah agreed, it will be harder to call it from inner class, but it's possible
approved